lib/fortnox/api/request_handling.rb

```diff
@@ -18,12 +18,13 @@ def validate_response(response)
 
         if response.headers['content-type'].start_with?('text/html')
           raise Fortnox::API::RemoteServerError,
-                "Fortnox API's response has content type \"text/html\" instead" \
-                'of requested "application/json".' \
+                "Fortnox API's response has content type \"text/html\" instead of requested \"application/json\"." \
                 'This could be due to invalid endpoint or when the API is down. ' \
                 "Body: #{response.body}"
         end
 
+        raise Fortnox::API::RemoteServerError, 'Unauthorized request' if response.code == 401
```

Comment:
Add one for 403 as well.

Reply:
And tell them what the actual error is 🙂 "No access token provided" and "Invalid or expired access toekn provided".